topics/spacevim/index.md

```diff
@@ -0,0 +1,21 @@
+---
+aliases: space-vim
+created_by: Wang Shidong
+display_name: SpaceVim
+github_url: https://github.com/SpaceVim/SpaceVim
+logo: spacevim.png
+released: January 26, 2017
+short_description: SpaceVim is a community-driven distribution of the vim editor that allows managing your plugins in layers.
+topic: spacevim
+url: https://spacevim.org/
+---
+
+SpaceVim is a distribution of the vim editor that's inspired by
+[spacemacs](https://github.com/syl20bnr/spacemacs). It manages collections
+of plugins in layers, which help collect related packages together to provide
+features. For example, the [python layer](http://spacevim.org/layers/lang/python/)
+collects [deoplete.nvim](https://github.com/Shougo/deoplete.nvim/), 
+[neomake](https://github.com/neomake/neomake) and [jedi-vim](https://github.com/davidhalter/jedi-vim)
```

Comment:
Can you add a comma here before `and`? See [the style guide](https://github.com/github/explore/blob/master/docs/styleguide.md#commas).

Reply:
Thanks. I have fix it. :)